docs/modules/ROOT/pages/release-notes/release-notes-2021-8.adoc

```diff
@@ -0,0 +1,69 @@
+# Release Notes - Payara Platform Community 5.2021.8
+
+
+### Supported APIs and Applications
+* Jakarta EE 8
+* Java EE 8 Applications
+* Jakarta EE 9
+* MicroProfile 4.1
+
+
+#### Breaking Changes
+
+##### Client Certificate Validation Checks
+
+After upgrading to Payara Community 5.2021.8,  a certificate that was accepted in earlier versions might be rejected now. It is no longer enough that the client certificates are included within the Payara Keystore. See [Certificate Expiration Validation :: Payara Community Documentation](https://docs.payara.fish/community/docs/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.html#_client-certificate-expiration-validator)
+
+### Improvements
+
+[[FISH-5687](https://github.com/payara/Payara/pull/5394)] Integrate HotSwap Agent in Payara Platform
```

Comment:
Links need to be moved in front of issue tags
```suggestion
[(https://github.com/payara/Payara/pull/5394)[FISH-5687]] Integrate HotSwap Agent in Payara Platform
```

Reply:
cc @debbiehoffman @rdebusscher - I'll update the internal docs for how this should be done